defaults/defaults.go

```diff
@@ -66,12 +66,15 @@ const (
 	ConnectivityCheckNamespace = "cilium-test"
 
 	ConnectivityCheckAlpineCurlImage = "quay.io/cilium/alpine-curl:v1.4.0@sha256:2550c747831ff575f2147149b088ea981c06f9b6bcd188756d1b82cc10997956"
+	ConnectivityPerformanceImage     = "quay.io/jtaleric/iperf3:latest"
```

Comment:
Would be nice to use a tagged release with image digest for this image as well. That way we can make image version updates in `cilium-cli` explicit via a PR and would catch potential issues in CI.